FXSkins/src/main/resources/com/pixelduke/control/skin/fx-skins.css

```diff
@@ -47,23 +47,31 @@ TextField {
     -fx-skin: "com.pixelduke.control.skin.FXTextFieldSkin";
 }
 
-TextField {
-    -right-button-visible: true;
+.text-field {
+    -fx-padding: 0.3333333em 3em 0.3333333em 0.5833333em;
 }
 
-TextField > .right-button > .right-button-graphic {
+.text-field > .right-button > .right-button-graphic {
     -fx-shape: "M221.738,305.873l6.135,6.16l-2.875,2.863l-6.135-6.159l-6.263,6.237l-2.864-2.875l6.263-6.238l-6.177-6.202l2.875-2.863l6.177,6.201l6.244-6.22l2.864,2.876L221.738,305.873z";
 
     -fx-padding: 0.416667em;
-    -fx-background-color: #333;
+    -fx-background-color: #111;
 }
 
-TextField > .right-button:hover {
+.text-field > .right-button {
```

Comment:
We can't change this to ".text-field". There are reasons why "Textfield" is being used, namely there is a corner case where this will be a problem (check comment on base.css of JMetro).

Now that I'm looking at this, the comment on JMetro that says why we should use "TextField" should probably be added here as well to prevent regressions.

Reply:
I changed it to `.text-field` beause it also applies to the password textfield. If it does cause issues we need to copy code.